crates/oxc_language_server/src/linter/snapshots/fixtures_linter_cross_module_debugger.ts.snap

```diff
@@ -0,0 +1,4 @@
+---
+source: crates/oxc_language_server/src/linter/tester.rs
+---
+No diagnostic reports
```

Comment:
This is buggy. I expect that the default configuration rule `eslint/no-debugger` is reporting.
Feel free to fix it. 

I am creating other tests for nested configurations, before trying to merge the Runtime PR.